README.md

```diff
@@ -1,6 +1,6 @@
-# This project has been moved to the TypeScript ESLint organization -> https://typescript-eslint.io
+# Deprecated: TypeScript ESLint Parser
 
-The `typescript-eslint-parser` npm package will not receive any future updates, instead please follow along on the new monorepo, where the project will be released under the name `@typescript-eslint/parser`.
+**Important:** This repository is no longer maintained and `typescript-eslint-parser` will not receive any future updates. There is an actively maintained fork of this project available at https://typescript-eslint.io and published on npm as `@typescript-eslint-parser`.
```

Comment:
Per above, here's a suggested change to make this easier to tweak:

```suggestion
**Important:** This repository is no longer maintained and `typescript-eslint-parser` will not receive any future updates. There is an actively maintained fork of this project available at https://typescript-eslint.io and published on npm as `@typescript-eslint/parser`.
```

Reply:
Thank you, sir.